id.html

```diff
@@ -0,0 +1,238 @@
+            <a href="https://github.com/xTaiwanPingLord">TW-xTaiwanPingLord#9960</a>
+          </div>
+        </li>
+        <li class="dropdown-menu">
+          <span>Rilis
+            <i class="iconfont icon-xiahuasanjiao"></i>
+          </span>
+          <div>
+            <a href="https://github.com/Bkebi-Group/Bkebi-GC-Release/releases/latest"><img
+                src="https://img.shields.io/github/v/release/Bkebi-Group/Bkebi-GC-Release?style=for-the-badge">
+            </a>
+          </div>
+        </li>
+      </ul>
+    </nav>
+    <div id="banner">
+      <div id="banner-text">
+        <div>
+          <h1>Bkebi GC</h1>
+          <a href="https://github.com/Bkebi-Group/Bkebi-GC-Release/releases/latest">
+            <img src="https://img.shields.io/github/v/release/Bkebi-Group/Bkebi-GC-Release?style=for-the-badge">
+          </a>
+        </div>
+        <div>
+          <h1>Bkebi Genshin Cheat
+            <br>
+            <a href="https://github.com/Bkebi-Group/Bkebi-GC-Release/releases/latest"><img
+                src="https://img.shields.io/github/v/release/Bkebi-Group/Bkebi-GC-Release?style=for-the-badge"></a>
+            <a href="https://github.com/Bkebi-Group/Bkebi-GC-Release/releases"><img
+                src="https://img.shields.io/github/downloads/Bkebi-Group/Bkebi-GC-Release/total.svg?style=for-the-badge"></a>
+            <a href="https://discord.gg/bkebi"><img
+                src="https://img.shields.io/discord/1026295403282436097?label=Discord&logo=discord&style=for-the-badge&color=blueviolet"></a>
+          </h1>
+        </div>
+        <div>
+          <a id="download-button" class="btn"
+            href="https://github.com/Bkebi-Group/Bkebi-GC-Release/releases/latest">Dapatkan Sekarang
+            <i class="iconfont icon-xiazai"></i>
+          </a>
+          <a id="documentation-link" class="btn" href="docs/index.html" target="_blank">Dokumentasi 
+            <i class="iconfont icon-wendang"></i>
+          </a>
+        </div>
+      </div>
+    </div>
+    <a id="down-arrow" href="#kali-highlights">
+      <div>
+      </div>
+    </a>
+  </header>
+  <section id="kali-highlights">
+    <div>
+      <div>
+        <h1>Gratis dan Canggih</h1>
+      </div>
+    </div>
+    <h3>Menghabiskan lebih sedikit energi dan waktu, membantu menyelesaikan pekerjaan mu, memperbaiki perngalaman game anda, dan tidak lagi menderita dengan akun yang burik.</h3>
+    <ul>
+      <li>
+        <a href="javascript:;">
+          <h4>
+            <i class="iconfont icon-youxiyouxisheji"
+              style="margin-right: 0.4em;line-height: 1;font-size: 25px;vertical-align: bottom;">
+            </i>Player
+          </h4>
+        </a>
+        <p>Mode serangan ganda, untuk menghindari terdeteksi oleh server dari damage yang tinggi. Hapus cooldown, ledakan elemen tidak perlu menunggu cooldown, karena variasi permainan, server tidak akan memeriksanya. Mode tak terkalahkan probabilitas yang sudah di sesuaikan.</p>
```

Comment:
change it to `Mode serangan multiplier, untuk menghindari kerusakan yang tinggi dan dicatat oleh server. Menghapus cooldown, jadi tidak perlu mengumpulkan energi dan menunggu cooldown Elemental Burst. Dalam hal ini, server tidak memeriksa perilaku pemain dalam keragaman gameplay. God Mode tidak akan terkena serangan tertentu, Anda dapat mengatur kondisi, dan dapat mengatur peluang terkena hit. Memungkinkan pemain untuk lebih leluasa dalam bermain. Mode stamina tak terbatas, dapat bebas bergerak, dan multipel animasi.`